modules/apps/portal-search/portal-search-web/src/test/java/com/liferay/portal/search/web/internal/facet/display/context/AssetTagsSearchFacetDisplayContextTest.java

```diff
@@ -181,8 +183,101 @@ public void testOneTermWithPreviousSelection() throws Exception {
 			assetTagsSearchFacetDisplayContext.isRenderNothing());
 	}
 
```

Comment:
@felipelorenz @joshuacords  Maybe could be a good idea to add two different asserts in each test, the first one will be to test the ordering without any equal frequency or equal term name, and the second one will be like the tests are already asserting, what you guys think about that? 

That way we will have more consistent tests

Reply:
Yes, we should be testing the defaults as well - great point Gustavo!